src/components/header.tsx

```diff
@@ -1,109 +1,237 @@
 import React from "react";
-import { IoIosArrowBack } from "react-icons/io";
-
 import {
   advisoryLogo,
   advisoryMentorshipLogo,
   advisoryMentorshipPartners,
 } from "../assets";
 
 import "./header.css";
+//for responsive drawer
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Divider from "@mui/material/Divider";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import Toolbar from "@mui/material/Toolbar";
+import MenuIcon from "@mui/icons-material/Menu";
+import ListItemButton from "@mui/material/ListItemButton";
 
-const NavBar = () => (
-  <div className="nav">
-    <a className="nav-back-to-main-site" href="https://advisory.sg">
-      <IoIosArrowBack className="nav-arrow-icon" />
-      back to main site
-    </a>
-    <img className="nav-logo" src={advisoryLogo} alt="Advisory" />
-  </div>
-);
+const drawerWidth = 240;
 
-const Header = () => (
-  <div className="header">
-    <NavBar />
-    <div className="header-bottom">
-      <img
-        className="header-mentorship-logo"
-        src={advisoryMentorshipLogo}
-        alt="Advisory Mentorship Programme"
-      />
-      <img
-        className="header-mentorship-logo"
-        src={advisoryMentorshipPartners}
-        alt="Advisory Mentorship Programme Partners"
-      />
-      <div className="header-mentorship-intro">
-        <p>
-          The Advisory Mentorship Programme pairs students with working
-          professionals in their fields of interest on a 1-1 basis. Over the
-          course of four months, from September to December 2021, mentors give
-          an hour each month to meet with their mentee. In its first run in
-          2020, the programme delivered over 2000 hours of mentorship to 694
-          students with the involvement of over 562 mentors. This year, for Wave
-          1 of the programme which is taking place from June to September 2021,
-          nearly 900 working professionals came aboard as mentors for 1,104
-          mentees. For Wave 2, we are pleased to have over 400 mentors,
-          including from 10 mentorship partners, whose fields of expertise range
-          across 48 different industries.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://advisorysg.typeform.com/to/HQ8nWq2r#source=mentorsite"
+interface Props {
+  /**
+   * Injected by the documentation to work in an iframe.
+   * You won't need it on your project.
+   */
+  window?: () => Window;
+}
+//drawer
+function ResponsiveDrawer(props: Props) {
+  const { window } = props;
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const drawer = (
```

Comment:
Please convert this into a proper React component, i.e. a function as in https://reactjs.org/docs/components-and-props.html, preferably an arrow function e.g. `const Drawer = () => ( ... );`.

Also, a more appropriate name might be `LinkList`.